src/prediqt.ts

```diff
@@ -321,20 +303,14 @@ export class Prediqt {
     /**
      * Transfer shares between users
      */
-    public async transferShares(from: string, to: string, shares: number, sharetype: boolean, marketId: number): Promise<any> {
+    public async transferShares(data: TransferShares): Promise<any> {
```

Comment:
I think we should use our own transformation for variable names, since in the  blockchain side we have market_id but we should follow our camelCase conventions otherwise its gonna be a bad mix when someone uses our prediqt-js api.